platformio.ini

```diff
@@ -123,7 +123,7 @@ lib_deps =
 
 [radiolib_base]
 lib_deps =
-	jgromes/RadioLib@7.1.0
+	https://github.com/jgromes/RadioLib.git#b2b4c9e0f73bd9d5a435b291ee27574ee9c2f69f
```

Comment:
As above: do we need the specific reference?

Reply:
Yes, there's a need for a fix that isn't released yet